gemini/use-cases/retail/product_attributes_extraction.ipynb

```diff
@@ -257,6 +256,29 @@
     "        return \"image/jpeg\"\n",
     "\n",
     "\n",
+    "def load_bytes_from_uri(image_uri: str) -> bytes:\n",
+    "    \"\"\"Load image bytes from a remote or local file URI.\n",
+    "\n",
+    "    Args:\n",
+    "        image_uri (str): The uri or the local file path for the image.\n",
+    "\n",
+    "    Returns:\n",
+    "        The object/image bytes.\n",
+    "    \"\"\"\n",
+    "    image_bytes = None\n",
+    "    if image_uri.startswith(\"http://\") or image_uri.startswith(\"https://\"):\n",
+    "        response = requests.get(image_uri)\n",
+    "        if response.status_code == 200:\n",
+    "            image_bytes = response.content\n",
+    "        else:\n",
+    "            print(\n",
+    "                f\"Fetch image failed for {image_uri}, status code: {response.status_code}\"\n",
+    "            )\n",
+    "    else:\n",
+    "        image_bytes = open(image_uri, \"rb\").read()\n",
+    "    return image_bytes\n",
+    "\n",
+    "\n",
```

Comment:
To make the rest of the code simpler, I wonder if you could modify this function to be `load_part()` and have it do the check for `gs://` as well.

Reply:
done.